### PR TITLE
Provides upstream ACIS API call for 2 years worth of data

### PR DESCRIPTION
**Pull request by BobTorgerson**
This PR pulls data from the online ACIS API to collect 2 years worth of data to be displayed on the graph. In addition, the graph shows a loading icon while waiting for the upstream data to be collected and returned in a proper format in a Pandas DataFrame. 

Added ACCAP favicon image to site.

Added Python Black to development pipenv for desired code formatting.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+.idea/*
```
Comment (brucecrevensten) [inline .gitignore:6]: What's this for?

File: Pipfile [hunk 2]
```diff
@@ -12,9 +12,14 @@ plotly = "*"
 dash_dangerously_set_inner_html = "*"
 pandas = "*"
 beaker = "*"
+scipy = "*"
 
 [dev-packages]
 flask = "*"
+black = "*"
```
Comment (brucecrevensten) [inline Pipfile]: Usually I run this from a global install.  Is that wrong?  I had really high friction getting my editor to use the local `pipenv` to integrate with Black.  We can discuss if that makes sense or not.

File: Pipfile [hunk 3]
```diff
@@ -12,9 +12,14 @@ plotly = "*"
...
 
 [requires]
 python_version = "3.6"
+
+[pipenv]
+allow_prereleases = true
```
Comment (brucecrevensten) [inline Pipfile]: Why is this needed?

File: data.py [hunk 4]
```diff
@@ -26,30 +26,193 @@
 cache_opts = {"cache.type": "memory"}
 
 cache = CacheManager(**parse_cache_config_options(cache_opts))
-data_cache = cache.get_cache("api_data", type="memory", expire=CACHE_EXPIRE)
+# data_cache = cache.get_cache("statewide_temp_index", type="memory", expire=CACHE_EXPIRE)
```
Comment (brucecrevensten) [inline data.py]: Remove commented code unless this needs to be here for some reason.

File: gui.py [hunk 5]
```diff
@@ -4,13 +4,11 @@
...
 
 import os
-import datetime
 import dash_core_components as dcc
 import dash_html_components as html
 import dash_dangerously_set_inner_html as ddsih
 import luts
-import data
-import charts
+from datetime import datetime
```
Comment (brucecrevensten) [inline gui.py]: Move to line 7 (pylint C0411)

**Review by brucecrevensten — CHANGES_REQUESTED** [review]
I tested the actual output by checking the last-available graphic from the ACCAP website (which was old, but whatever) against the matching date range.  It's not an exact match but we expected that (the old code had some very station-specific adjustments which we removed, I've discussed that with Rick).  So that looks good.

I wish the loading overlay was more... something.  But!  Let's not get hung up on that!  This is a great first draft and we can improve it over time.  Maybe there are some easy tweaks, but let's do them in a separate branch.

One or two other things to check, then it looks good!

File: .gitignore (hunk 1 above)
Comment (BobTorgerson) [reply .gitignore:6]: This is for my PyCharm IDE, I added it so I wouldn't see it in my Git output.

File: Pipfile (hunk 2 above)
Comment (BobTorgerson) [reply Pipfile]: I'm going to remove Python Black as we have decided to install this globally as it is not really a single project's package, but should be used for code formatting across all Python applications.

File: Pipfile (hunk 3 above)
Comment (BobTorgerson) [reply Pipfile]: This was needed to allow for Python Black to be installed, as it has yet to have an official "production" release, and only provides pre-releases for its multiple versions. Removing this along with Python Black.

**Review by brucecrevensten — APPROVED** [review]
